### PR TITLE
update hebungsraten layer

### PR DESCRIPTION
**Pull request by daguer**

### CODE REVIEW

**Comment by rfrancel** [comment]
@gjn

File: chsdi/locale/de/LC_MESSAGES/chsdi.po [hunk 1]
```diff
@@ -3824,6 +3824,10 @@ msgstr "Eignung"
 msgid "ch.bfe.solarenergie-eignung-daecher.neigung"
 msgstr "Neigung [°]"
 
+#. TODO
+msgid "ch.bfe.solarenergie-eignung-daecher_overview"
+msgstr "ch.bfe.solarenergie-eignung-daecher_overview"
+
```
Comment (loicgasser) [inline chsdi/locale/de/LC_MESSAGES/chsdi.po]: missing translation

File: chsdi/locale/en/LC_MESSAGES/chsdi.po [hunk 2]
```diff
@@ -3824,6 +3824,10 @@ msgstr "Suitability"
 msgid "ch.bfe.solarenergie-eignung-daecher.neigung"
 msgstr "Inclination [°]"
 
+#. TODO
+msgid "ch.bfe.solarenergie-eignung-daecher_overview"
+msgstr "ch.bfe.solarenergie-eignung-daecher_overview"
```
Comment (loicgasser) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: again

File: chsdi/locale/fi/LC_MESSAGES/chsdi.po [hunk 3]
```diff
@@ -3824,6 +3824,10 @@ msgstr "Eignung"
 msgid "ch.bfe.solarenergie-eignung-daecher.neigung"
 msgstr "Neigung [°]"
 
+#. TODO
+msgid "ch.bfe.solarenergie-eignung-daecher_overview"
+msgstr "ch.bfe.solarenergie-eignung-daecher_overview"
```
Comment (loicgasser) [inline chsdi/locale/fi/LC_MESSAGES/chsdi.po]: again

File: chsdi/models/vector/stopo.py [hunk 4]
```diff
@@ -1942,6 +1942,7 @@ class HebungsratenLine(Base, Vector):
     __table_args__ = ({'schema': 'geodaesie', 'autoload': False})
     __template__ = 'templates/htmlpopup/hebungsraten.mako'
     __bodId__ = 'ch.swisstopo.hebungsraten'
+    __queryable_attributes__ = []
```
Comment (loicgasser) [inline chsdi/models/vector/stopo.py]: why is that empty?

File: chsdi/locale/fr/LC_MESSAGES/chsdi.po [hunk 5]
```diff
@@ -3824,6 +3824,10 @@ msgstr "Aptitude"
 msgid "ch.bfe.solarenergie-eignung-daecher.neigung"
 msgstr "Inclination [°]"
 
+#. TODO
+msgid "ch.bfe.solarenergie-eignung-daecher_overview"
+msgstr "ch.bfe.solarenergie-eignung-daecher_overview"
```
Comment (loicgasser) [inline chsdi/locale/fr/LC_MESSAGES/chsdi.po]: again

File: chsdi/locale/it/LC_MESSAGES/chsdi.po [hunk 6]
```diff
@@ -3824,6 +3824,10 @@ msgstr "Aptitude"
 msgid "ch.bfe.solarenergie-eignung-daecher.neigung"
 msgstr "Inclination [°]"
 
+#. TODO
+msgid "ch.bfe.solarenergie-eignung-daecher_overview"
+msgstr "ch.bfe.solarenergie-eignung-daecher_overview"
```
Comment (loicgasser) [inline chsdi/locale/it/LC_MESSAGES/chsdi.po]: again

**Comment by loicgasser** [comment]
@daguer please address and I'll merge

File: chsdi/models/vector/stopo.py (hunk 4 above)
Comment (loicgasser) [inline chsdi/models/vector/stopo.py]: This is not needed anymore with https://github.com/geoadmin/mf-chsdi3

**Comment by oterral** [comment]
Translations are not related to this layer.

**Comment by oterral** [comment]
@loicgasser The 4 attributes are still displayed in the list of the queryable attributes in the query tool. So we let as it is for now.

File: chsdi/locale/de/LC_MESSAGES/chsdi.po (hunk 1 above)
Comment (loicgasser) [inline chsdi/locale/de/LC_MESSAGES/chsdi.po]: oui mais t'as te louper dans le rebase car cette couche n'est pas dans le empty.po.

**Comment by loicgasser** [comment]
@oterral this was to rebased it is now fixed.

**Comment by oterral** [comment]
thx @loicgasser

**Pull request merged**